Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -81,3 +81,19 @@
   categories:
   - UplinkImplants
+
+- type: listing
+  id: UplinkBloodDagger
+  name: uplink-blood-dagger-name
+  description: uplink-blood-dagger-desc
+  productEntity: BloodDagger
+  cost:
+    Telecrystal: 8
+  categories:
+  - UplinkWeapons
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)